gems/pending/VMwareWebService/MiqPbmInventory.rb

```diff
@@ -0,0 +1,32 @@
+require 'VMwareWebService/PbmService'
+
+module MiqPbmInventory
+  def pbm_initialize(server, username, password)
+    @pbm = PbmService.new(server, username, password) if @apiVersion >= '5.5'
```

Comment:
nitpick: how about rename ```@pbm``` to ```@pbm_service or @pbm_svc```?  That'd be easier for reader to tell this is the PbmService object (vs the ```pbm connection``` in ```PbmService```)